src/write/segy_write_append.jl

```diff
@@ -0,0 +1,16 @@
+export segy_write_append
+
+function segy_write_append(file::String, block::SeisBlock)
+
+    # Open buffer for writing
+    s = open(file, "r+")
+
+
+    # Write Data
+    ns,ntraces = size(block.data)
+    for t in 1:ntraces
+        write_trace(seekend(s), block, t)
```

Comment:
You shouldn't need `seekend` if you do `open(file, "a")` then the buffer should automatically be at the end and you can just write as it goes `write_trace(s, block, t)`

Reply:
Unless I remember it wrong, I do not believe you can append to SEGY file like that. you have to rewrite file header part on append.

Reply:
> You shouldn't need `seekend` if you do `open(file, "a")` then the buffer should automatically be at the end and you can just write as it goes `write_trace(s, block, t)`

I'll keep it by using the "r+" cause it seems that we need to check the fileheader at the beginning and use the read_fileheader.jl which changes the buffer.

Reply:
> Unless I remember it wrong, I do not believe you can append to SEGY file like that. you have to rewrite file header part on append.

Yeah, you are right. For this situation, the file header part should be same with each other and I just added this condition by using the function check_fileheader.jl.

Reply:
Doesn't file header have number of traces as one of the fields? I cannot recall.

Reply:
> Doesn't file header have number of traces as one of the fields? I cannot recall.

No, I checked the file header, it does not have the number of traces. It has the number of samples along each trace and this should be same on append.

Reply:
Thanks for checking and sorry for failing memory.

Reply:
> Thanks for checking and sorry for failing memory.

You are welcome and thanks for your comments.